src/cpp/client/ProxyClient.cpp

```diff
@@ -18,6 +18,8 @@
 #include <micrortps/agent/datareader/DataReader.hpp>
 #include <micrortps/agent/datawriter/DataWriter.hpp>
 #include <micrortps/agent/topic/Topic.hpp>
+#include <micrortps/agent/libdev/MessageDebugger.h>
```

Comment:
Encapsulate this include in `#ifdef VERBOSE_OUTPUT` to improve compilation time when not using verbose_output.